hw/chisel/src/main/scala/snax/streamer/DesignParams.scala

```diff
@@ -86,13 +89,28 @@ class StreamerParam(
     )
   }
 
-  val dataPathExtensionParam: Seq[HasDataPathExtension] =
+  val dataPathABExtensionParam: Seq[HasDataPathExtension] =
     (if (hasTranspose)
        Seq[HasDataPathExtension](
          HasTransposer
        )
      else
        Seq[HasDataPathExtension]())
+
+  val dataPathCExtensionParam: Seq[HasDataPathExtension] =
+    (if (hasCBroadcast)
```

Comment:
these are very gemm-specific parameters in the main streamer files, can we make this a bit more generic?